packages/mui-joy/src/ToggleButtonGroup/ToggleButtonGroupContext.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+
+interface IToggleButtonGroupContext {
+  onClick: (
+    event: React.MouseEvent<HTMLButtonElement>,
+    childValue: readonly string[] | string | number | undefined,
+  ) => void;
+  value?: string | (string | number)[] | null;
+}
+
+/**
+ * @ignore - internal component.
+ */
+const ToggleButtonGroupContext = React.createContext<IToggleButtonGroupContext>({
+  onClick: () => {},
+  value: null,
+});
```

Comment:
No need for default values. What's the purpose of having them?
```suggestion
const ToggleButtonGroupContext = React.createContext<IToggleButtonGroupContext>({});
```